src/Slider.js

```diff
@@ -3,39 +3,37 @@ import PropTypes from 'prop-types';
 import cx from 'classnames';
 
 class Slider extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.initSlider = this.initSlider.bind(this);
     this.fullscreenReset = this.fullscreenReset.bind(this);
     this.setActiveIndex = this.setActiveIndex.bind(this);
   }
 
   initSlider() {
-    const { indicators, interval, duration, height } = this.props;
-    this.instance = M.Slider.init(this._slider, {
-      indicators,
-      interval,
-      duration,
-      height
-    });
+    this.instance = M.Slider.init(this._slider, this.props.options);
   }
 
   /**
    * If the slider was not in fullscreen, the height is set as a style attribute
    * on the Slider element. When `.destroy()` is called, this attribute is not
    * removed, resulting in a fullscreen displayed incorrectly.
    */
-  fullscreenReset(was_fullscreen) {
-    if (!was_fullscreen && this.props.fullscreen) {
+  fullscreenReset(wasFullscreen) {
+    if (!wasFullscreen && this.props.fullscreen) {
       this.instance.el.removeAttribute('style');
       this.instance.el.childNodes[0].removeAttribute('style');
     }
   }
 
   setActiveIndex(activeIndex) {
-    if (this.props.indicators && activeIndex)
+    let { indicators } = this.props.options;
+    // In case this option is not defined, we assume true, as per default
+    let showIndicators = typeof indicators === 'undefined' || indicators;
+    if (showIndicators && activeIndex) {
```

Comment:
If `indicators` is undefined it should evaluate to true. `if (indicators && activeIndex)` if indicators is undefined, it evaluates to false.

Reply:
I see, but typeof indicators === 'undefined' || indicators says that "if indicators is undefined or not, we will do it"

if it is undefined or not will make no difference in this case cause it will always return true.

See if you agrees with me: https://codepen.io/onluiz/pen/KBjGYE?editors=1010

If not, please explain cause a may be wrong :nerd_face: 

Also, thanks for your help!

Reply:
The `indicators` variable is a boolean. So if `indicators = false` the expression is false.